### ISSUE TITLE
Does not work if website overrides Date.prorotype.toString

### ISSUE THREAD

**Issue by mwisnicki** [violentmonkey/violentmonkey#1115]
### What is the problem?
Intranet website overrides Date.prototype.toString.
The override fails if called with radix argument (which is undocumented on MDN).
Somewhere in injected-web.js is a call to `Date.now().toString(36)`.

Apologies for lack of easy repro due to page being on intranet. If it's a big deal I can whip up something later.

There are two problems here:

1. Assuming `Date.now().toString(36)` indeed comes from this repo (I couldn't find it with github search), injected-web perhaps shouldn't rely on non-standard undocumented API.

2. It looks to me like a big architectural issue to rely on website environment for execution of some of extension code.
It looks like a website is basically able to do code injection into extension code by patching standard objects.
Is there a way for injected-web to somehow execute in pristine environment?

### Environment
- Browser: Chrome
- Browser version: 87.0.4280.66 (Official Build) (64-bit)
- Violentmonkey version: 2.12.7
- Violentmonkey [pre-release behavior](https://github.com/violentmonkey/violentmonkey/releases):
- OS: Windows 10

**Comment by mwisnicki** [comment]
For reference this is formatted minimized code from  injected-web.js:formatted:220:

```
        function o(e) {
            return (e || "") + Date.now().toString(36) + Math.floor(65536 * (1 + Math.random())).toString(16).slice(-4)
        }
```

`e` has value `"VMin"` if it helps. Looks to be a call to `getUniqId("VMin")`?

**Comment by tophf** [comment]
This has been fixed already in the [upcoming pre-release](https://github.com/violentmonkey/violentmonkey/releases).

**Comment by mwisnicki** [comment]
@tophf Thanks, that was a quick response! I'm assuming it's fixed via b50e2552c96c271b7e3549a4f75970bdf559a3db which would address problem 1.

I'm curious about the 2nd problem though. Do you think it could be a security issue?

**Comment by gera2ld** [comment]
@mwisnicki It could be but I don't think there is a better way to do this because we share one context in browser and that's how JavaScript works.

That being said, we are trying to keep the original methods for internal usage:

https://github.com/violentmonkey/violentmonkey/blob/6431222615e6eb98d228542c7ac5655e4a1a6463/src/injected/utils/helpers.js#L5-L13

But we cannot guarantee that the methods used by userscripts are the original ones.
